src/Mvc/Mvc.Core/src/ModelBinding/ReferenceEqualityComparer.cs

```diff
@@ -9,8 +9,6 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
 {
     internal class ReferenceEqualityComparer : IEqualityComparer<object>
```

Comment:
If you'd like to go the extra mile, there's a `https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Collections/Generic/ReferenceEqualityComparer.cs#L30` in the runtime now that should be drop-in replacement.

Reply:
I made this change in the browser 🤣

Reply:
Looks worthwhile though